### ISSUE TITLE
Tailwind error: slider-txt-rtl class does not exist

### ISSUE THREAD

**Issue by laurensiusadi** [vueform/slider#57]
Error message
```
The `slider-txt-rtl` class does not exist. If `slider-txt-rtl` is a custom class, make sure it is defined within a `@layer` directive.
```

Dependencies:
```
"@vueform/slider": "^2.0.9",
"vue": "^3.2.25",
```
Dev dependencies
```
"autoprefixer": "^10.4.2",
"postcss": "^8.4.6",
"sass": "^1.51.0",
"tailwindcss": "^3.0.18",
"typescript": "^4.5.4",
"vite": "^2.9.0",
```

Error only exist when using `tailwind.scss` import. It's not error when using `classes` directly.

**Comment by dzcpy** [comment]
Same issue, I'm using `vue 2` and css

**Comment by robertosanval** [comment]
Same issue here using `Vue 3`

**Comment by adamberecz** [comment]
Thanks for reporting - fixed in `2.1.6`.
